### DISCUSSION TITLE
How do I suppress "open Telegram app" dialog?

### DISCUSSION THREAD

**Discussion by Nakilon** [rubycdp/ferrum#361] [Q&A]
I want to suppress the following dialog

![image](https://github.com/rubycdp/ferrum/assets/2870363/7a6b02f5-b243-4e27-8783-e3d82a24e388)

I found this https://stackoverflow.com/a/60347389/322020, now how do I apply this? How do I pass the "prefs"?
This dialog appears when I visit t.me links.

**Comment by route** [comment]
Needs investigation, depending on what selenium does with `setCapability` call, it might be a file that is set as flag or a flag.
